src/containers/Moods.js

```diff
@@ -2,71 +2,105 @@ import React from 'react';
 import { connect } from 'react-redux';
 import Controls from '../components/controls/Controls';
 import Face from '../components/face/Face';
-import store from '../store';
 import PropTypes from 'prop-types';
 import Timer from '../components/timer/timer';
 import StartButton from '../components/startButton/StartButton';
-
-const actions = [
-  { name: 'DRINK_COFFEE', text: 'Drink Coffee', stateName: 'coffees' },
-  { name: 'EAT_SNACK', text: 'Snack', stateName: 'snacks' },
-  { name: 'TAKE_NAP', text: 'Nap', stateName: 'naps' },
-  { name: 'STUDY', text: 'Study', stateName: 'studies' },
-];
-
-export const isTired = state => state.coffees < 1 && state.naps < 1;
-export const isHyper = state => state.coffees > 3;
-export const isEducated = state => state.studies > 2;
-export const isHungry = state => state.snacks < 1;
-
-export const getFace = state => {
-  if(isTired(state) && isHungry(state)) return '😠';
-  if(isHyper(state) && isHungry(state)) return '🙀';
-  if(isTired(state)) return '😴';
-  if(isHyper(state)) return '🙀';
-  if(isEducated(state)) return '🤓';
-  if(isHungry(state)) return '😡';
-
-  return '😀';
-};
-
-const Moods = ({ dispatch, count, hasStarted }) => {
-  const state = store.getState();
-  const face = getFace(state);
-  const controlActions = actions.map(action => ({
-    ...action,
-    count: state[action.stateName]
-  }));
-
+import { handleSelectionAction } from '../actions/handleSelectionAction';
+import { toggleStartAction } from '../actions/toggleStartAction';
+import { decrementCounterAction } from '../actions/decrementCounterAction';
+import { restartStateAction } from '../actions/restartStateAction';
+import { getCoffees, getSnacks, getNaps, getStudies } from '../selectors/buttonStateSelectors';
+import { getTimerCount } from '../selectors/getTimerCount';
+import { getHasStarted } from '../selectors/getHasStarted';
+import { getFace } from '../selectors/getFace';
+import { getActions } from '../selectors/getActions';
+import ResetButton from '../components/resetButton/ResetButton';
+import Sidebar from '../components/Sidebar/Sidebar';
+import { getSavedGames } from '../selectors/getSavedGames';
+import SaveButton from '../components/saveButton/SaveButton';
+import { saveGameAction } from '../actions/saveGameAction';
+import styles from './Moods.css';
```

Comment:
def could put some selectors and actions in the same files and not have so many folders and files to import from. same thing for the reset and save buttons, could put them in the controls folder. file tree is a bit busy.